.github/workflows/ci.yml

```diff
@@ -303,7 +303,7 @@ jobs:
       - uses: maxim-lobanov/setup-xcode@v1
         with:
           xcode-version: '16.2'
-        if: ${{ matrix.platform == 'ipa' }}
+        if: ${{ runner.os == 'macos' }}
```

Comment:
`firebase_core` requires Xcode version to be at least 16.2 - https://firebase.google.com/docs/ios/setup#prerequisites

Otherwise macOS builds fails to compile Firebase's Swift code:

```
/Users/runner/work/messenger/messenger/macos/Pods/FirebaseCoreInternal/FirebaseCore/Internal/Sources/Utilities/UnfairLock.swift:26:42: error: expected ',' separator
  public init(_ value: consuming sending Value) ***
```